LayoutTests/http/tests/lockdown-mode/svg-with-unsupported-font-does-not-crash.html

```
@@ -0,0 +1,23 @@
+
+function test() {
+    if (window.FileSystemHandle == undefined)
+    	testPassed("Confirmed Lockdown Mode process");
+    else
+        testFailed("This is NOT a Lockdown Mode process.");
+
+    testPassed("The SVG loaded without crashing.");
+    finishJSTest();
+}
+</script>
+</head>
```

HTMLHint (1.9.2)

[error] <title> must be present in <head> tag.

(title-require)